.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: DOCS
+    url: https://gofrp.org/en/docs/
```

Comment:
We haven't provide a English version here, just link repo README page?

Reply:
It was an oversight on my part, and I'm terribly sorry about it.